StudySwift/StudySwiftUITests/StudySwiftUITests.swift

```diff
@@ -22,12 +22,77 @@ final class StudySwiftUITests: XCTestCase {
         // Put teardown code here. This method is called after the invocation of each test method in the class.
     }
 
-    func testExample() throws {
+    func testSignUp() throws {
         // UI tests must launch the application that they test.
         let app = XCUIApplication()
         app.launch()
-
-        // Use XCTAssert and related functions to verify your tests produce the correct results.
+        
+        let emailField = app.textFields["Email"]
+        XCTAssertTrue(emailField.exists)
+        emailField.tap()
+        emailField.typeText("example@domain.com")
+        
+        let passwordField = app.secureTextFields["Password"]
+        XCTAssertTrue(passwordField.exists)
+        passwordField.tap()
+        passwordField.typeText("password")
```

Comment:
I'd change this password to one that contains special characters and numbers in it. For example, P@ssword12356.